zokrates_core/src/static_analysis/propagation.rs

```diff
@@ -718,7 +720,9 @@ impl<'ast, T: Field> Folder<'ast, T> for Propagator<'ast, T> {
             UExpressionInner::Not(box e) => {
                 let e = self.fold_uint_expression(e).into_inner();
                 match e {
-                    UExpressionInner::Value(v) => UExpressionInner::Value((!v) & 0xffffffff),
+                    UExpressionInner::Value(v) => {
```

Comment:
small format fix: these curly braces are not required

Reply:
hmm interesting I don't remember adding those, it seems to be added by `cargo fmt`

Reply:
interesting, can you confirm that?

Reply:
yep, just tested it